app/models/event1/MembersOrEmployersSummary.scala

```diff
@@ -24,7 +24,7 @@ import play.api.libs.functional.syntax._
 import play.api.libs.json.Reads._
 import play.api.libs.json.{Format, JsPath, Json, Reads}
 
-case class MembersOrEmployersSummary(name: String, unauthorisedPaymentValue: BigDecimal)
+case class MembersOrEmployersSummary(name: String, unauthorisedPaymentValue: BigDecimal, memberStatus: Option[String])
```

Comment:
I think this should be an enumerated type. That way we avoid repeating string values in the code